tools/ClassGen/NodeGen.cpp

```diff
@@ -315,6 +315,9 @@ int main(int argc, char **argv) {
                     "Weights[0] * Slice(0) + Weights[1] * Slice(1) + ... "
                     "It implies that len(Weights) == len(Indices).");
 
+  BB.newNode("IsNaN").addInput("Input").addResultFromCtorArg().setDocstring(
```

Comment:
nit: please use `// clang-format off` and `// clang-format on` here -- see the `Log` node also in this file